src/test/java/com/epam/tc/hw3/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,63 @@
+package com.epam.tc.hw3.ex1;
+
+import com.epam.tc.hw3.AbstractChromeTest;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.testng.annotations.Test;
+import pages.MainPage;
+
+public class Exercise1Test extends AbstractChromeTest {
+
+    private static final int COUNT_OF_IMAGES_ON_MAIN_PAGE = 4;
+    private static List<String> textUnderImages = List
+            .of("To include good practices\nand ideas from successful\nEPAM project",
+                    "To be flexible and\ncustomizable",
+                    "To be multiplatform",
+                    "Already have good base\n(about 20 internal and\nsome external projects),\nwish to get more…");
+
+    @Test(groups = {"HW3"})
+    public void exercise1() {
+        SoftAssertions softly = new SoftAssertions();
+        MainPage mainPage = new MainPage(driver);
```

Comment:
Let's declare mainPage variable in base class. For example: 
protected MainPage mainPage;
...and in your setup() method you could say: mainPage = new MainPage(driver);
So you could remove line 21 from your test.
Try this approach for all your pages.